shell/monitor/monitor.sh

```diff
@@ -39,22 +39,34 @@ _monitor_set_experiment_info_func(){
   _monitor_experiment_info_func=( "$@" )
 }
 
+_monitor_get_code(){
+  local output=$1
+  echo "$output" | tail -n 1
+}
+
+_monitor_get_body(){
+  local output=$1
+  echo "$output" | head -n -1
+}
+
+_monitor_do_curl(){
+
```

Comment:
Minor point about consistency, we don't always start/end a function with a blank line. We might as well be consistent, at least within this file.

Reply:
I find it more readable, so I put a blank line at the beginning of every function.